drivers/clock_control/clock_control_renesas_cpg_mssr.h

```diff
@@ -7,6 +7,68 @@
 #ifndef ZEPHYR_DRIVERS_RENESAS_RENESAS_CPG_MSSR_H_
 #define ZEPHYR_DRIVERS_RENESAS_RENESAS_CPG_MSSR_H_
 
+#include <zephyr/spinlock.h>
+#include <zephyr/sys/sys_io.h>
+#include <zephyr/drivers/clock_control.h>
+
+struct cpg_clk_info_table {
+	uint32_t domain;
+	uint32_t module;
+	mem_addr_t offset;
+	uint32_t parent_id;
+
+	uint32_t in_freq;
+	uint32_t out_freq;
+
+	enum clock_control_status status;
+
+	struct cpg_clk_info_table *parent;
+	struct cpg_clk_info_table *children_list;
+	struct cpg_clk_info_table *next_sibling;
+};
+
+struct rcar_cpg_mssr_data {
+	mem_addr_t base_addr;
+	struct cpg_clk_info_table *clk_info_table[2];
+	const uint32_t clk_info_table_size[2];
+	struct k_spinlock lock;
+
+	uint32_t (*get_div_helper)(uint32_t reg, uint32_t module);
+	int (*set_rate_helper)(uint32_t module, uint32_t *div, uint32_t *div_mask);
+};
+
+#define RCAR_CPG_NONE -1
```

Comment:
As I can see, this value is always assigned and compare to unsigned variables, maybe we will choose another value for this?

Reply:
+1